code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -14,14 +14,12 @@ bzformation = 4
 freonformation = 5
 stimformation = 6
 nobliumformation = 7
-hexane_plasma_filtering = 8
-zauker_decomp = 9
-healium_formation = 10
-proto_nitrate_formation = 11
-zauker_formation = 12
-halon_formation = 13
-hexane_formation = 14
-proto_nitrate_response = 15 - 18
+zauker_decomp = 8
+healium_formation = 9
+proto_nitrate_formation = 10
+zauker_formation = 11
+halon_formation = 12
+proto_nitrate_response = 13 - 18
 fusion = 19
 metallic_hydrogen = 20
 nobiliumsuppression = INFINITY
```

Comment:
Priority should really be a define at this point, I'll do that tomorrow TM, but in the mean time, you should update up to fusion.

Reply:
i did??

Reply:
Ah you're right, Misread some things